recipe/meta.yaml

```diff
@@ -11,21 +11,32 @@ source:
   patches:
     # Can't seem to avoid /usr/local/ install without patch
     - qwtconfig.pri.patch
+    # build in release mode
+    - windows.patch
 
 build:
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py>=35]
   number: 2
-  skip: true  # [win]
+  skip: true  # [win and py36]
```

Comment:
It's to avoid 2 builds withe the same visual studio runtime (py35 and py36 use the same vc14 runtime).